miqi/skills/qraft-workflowspec-export/scripts/upload_run.py

```
@@ -0,0 +1,205 @@
+                with file_path.open("rb") as fh:
+                    resp = client.post(
+                        f"{base_url}/oauth2/dataUpload",
+                        headers={"Authorization": f"Bearer {access_token}"},
+                        files={"file": (file_path.name, fh, "application/json")},
+                    )
+                if resp.status_code == 0:
+                    raise httpx.TransportError("empty response (HTTP 000)")
+                body_text = resp.text
+                code, message = classify_response(resp, body_text)
+                if code == "OK" or resp.status_code in (400, 401, 403):
+                    # 业务确定性结果不重试；5xx/未知也直接分类返回（重试仅针对网络层）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)